### PR TITLE
Manage studies

### PR DESCRIPTION
**Pull request by jaserud**
Related issues: https://github.com/cancogen-virus-seq/roadmap/issues/21

This PR  basically added the manage studies page, its modals (plus form stuff) and the methods to fetch data from the svc

### CODE REVIEW

File: components/pages/401.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+import { ReactElement } from 'react';
+
+import { ErrorPageLayout } from '../PageLayout';
+import DMSAdminContact from '../DMSAdminContact';
+
+const Error401 = (): ReactElement => {
+  return (
+    <ErrorPageLayout errorTitle="401: Unauthorized" subtitle="Error 401 - Unauthorized">
+      You are curretnly not authorized to view the page you requested. If the problem persists,
```
Comment (caravinci) [inline components/pages/401.tsx:30]:
```suggestion
      You are currently not authorized to view the page you requested. If the problem persists,
```

File: components/UserDropdown.tsx [hunk 2]
```diff
@@ -176,6 +176,13 @@ const UserDropdown = ({ className }: { className?: string }): ReactElement => {
             }
           `}
         >
+          {userIsCollaborator && (
```
Comment (jaserud) [inline components/UserDropdown.tsx:179]: changed order

File: components/pages/studies/DeleteSubmitterModal.tsx [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+      onActionClick={onSubmit}
+      actionText="Remove"
+      closeText="Cancel"
+      title={'Remove Confirmation'}
+    >
+      <p
+        css={css`
+          padding-right: 7px;
+          padding-left: 7px;
+          ${defaultTheme.typography.baseFont}
```
Comment (jaserud) [inline components/pages/studies/DeleteSubmitterModal.tsx:54]: updated typography

File: components/pages/studies/StudiesTable.tsx [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+import GenericTable from '../../GenericTable';
+import { Bin } from '../../theme/icons';
+import defaultTheme from '../../theme/index';
+
+const getTableStyle = (theme: typeof defaultTheme) => css`
+  & tbody {
+    tr td {
+      vertical-align: top;
+      text-align: left;
+      border-top: 1px solid ${theme.colors.accent};
```
Comment (jaserud) [inline components/pages/studies/StudiesTable.tsx:36]: table styling update

**Review by jaserud — COMMENTED** [review]
updated types and rename collaborator to curator

**Review by caravinci — APPROVED** [review]
🦾 Squash that thing!

**Pull request merged**